### ISSUE TITLE
missing lwt constraint in the opam file

### ISSUE THREAD

**Issue by jchanco** [mirage/ezjsonm#18]
Building with Lwt support fails with "Error: Unbound module Lwt.Infix"
To reproduce:

```
 > ./configure --enable-lwt
 > make
```

Output is:

```
ocaml setup.ml -build 
Finished, 1 target (0 cached) in 00:00:00.
File "_tags", line 28, characters 22-34:
Warning: the tag "pkg_alcotest" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
File "_tags", line 37, characters 23-35:
Warning: the tag "pkg_alcotest" is not used in any flag or dependency declaration, so it will have no effect; it may be a typo. Otherwise you can use `mark_tag_used` in your myocamlbuild.ml to disable this warning.
+ /Users/juan.chanco/.opam/4.02.3/bin/ocamlfind ocamlc -c -g -annot -bin-annot -I lib -package hex -package jsonm -package lwt -package sexplib -package uutf -I lib -o lib/ezjsonm_lwt.cmo lib/ezjsonm_lwt.ml
File "lib/ezjsonm_lwt.ml", line 17, characters 5-14:
Error: Unbound module Lwt.Infix
Command exited with code 2.
Compilation unsuccessful after building 12 targets (0 cached) in 00:00:00.
E: Failure("Command ''/Users/juan.chanco/.opam/4.02.3/bin/ocamlbuild' lib/ezjsonm.cma lib/ezjsonm.cmxa lib/ezjsonm.a lib/ezjsonm.cmxs lib/ezjsonm-lwt.cma lib/ezjsonm-lwt.cmxa lib/ezjsonm-lwt.a lib/ezjsonm-lwt.cmxs -tag debug' terminated with error code 10")
make: *** [build] Error 1
```

Alternately, installing with opam fails similarly:

```
> opam install lwt ezjsonm
```

![screen shot 2015-11-03 at 11 02 11 am](https://cloud.githubusercontent.com/assets/14990174/10915006/76b80142-821a-11e5-9f2d-ac521de27562.png)

**Comment by avsm** [comment]
what version of lwt do you have installed? this needs quite a recent one.

**Comment by jchanco** [comment]
2.4.5 from opam.

**Comment by jchanco** [comment]
It seems the Lwt.Infix module doesn't exist yet in that version. Unfortunately, I was unsuccessful pining Lwt in opam (`opam pin add lwt --dev-repo`)

**Comment by jchanco** [comment]
I successfully pinned lwt to 2.5.0, can't get ezjsonm to install on top of that via opam

**Comment by jchanco** [comment]
Ok, sorry, one last thing. This is on OSX (10.9.5), where the default version of lwt in opam seems to be 2.4.5, Ezjsonm installs just fine with lwt on linux (debian-jessie) where lwt seems to default to 2.5.0

**Comment by jchanco** [comment]
OK, sorry. Pilot error. After re-initing opam, the default version of lwt is 2.5.0 and ezjsonm installs with no problems.

**Comment by samoht** [comment]
Seems that there is a missing constraint in  https://github.com/ocaml/opam-repository/blob/master/packages/ezjsonm/ezjsonm.0.4.2/opam#L26

**Comment by djs55** [comment]
This was fixed in 14ff73fdd6be53fdcfc12ef5d2faf43d1d96a4b9
